rest/model/admin_test.go

```diff
@@ -4,90 +4,246 @@ import (
 	"testing"
 
 	"github.com/evergreen-ci/evergreen"
-	"github.com/stretchr/testify/suite"
+	"github.com/mongodb/grip/send"
+	"github.com/stretchr/testify/assert"
 )
 
-type AdminModelSuite struct {
-	serviceSettings evergreen.Settings
-	apiSettings     APIAdminSettings
-	restartResp     *RestartTasksResponse
-	suite.Suite
-}
-
-func TestAdminModelSuite(t *testing.T) {
-	suite.Run(t, new(AdminModelSuite))
-}
-
-func (s *AdminModelSuite) SetupSuite() {
-	s.serviceSettings = evergreen.Settings{
-		Banner:      "banner text",
-		BannerTheme: evergreen.Information,
-		ServiceFlags: evergreen.ServiceFlags{
-			TaskDispatchDisabled: true,
-			MonitorDisabled:      true,
-			TaskrunnerDisabled:   true,
+var testSettings = evergreen.Settings{
```

Comment:
should we load this from a file or put it in a different scope? 

I'd almost prefer to wrap it in a function because I worry about mutable global values, even/particularly in tests.

Reply:
I put this in its own function